.idea/runConfigurations/Import.xml

```diff
@@ -16,8 +16,7 @@
       <option name="taskNames">
         <list>
           <option value="run" />
-          <option value="run" />
-          <option value="--args=&quot;import&quot;" />
+          <option value="--args=&quot;--config /etc/entitlementcard/config.yml --project-id bayern.ehrenamtskarte.app import&quot;" />
```

Comment:
These run configs are not platform independent. They should run without changing `/etc`.